hastexo-guacamole-client/guacamole/client.py

```diff
@@ -0,0 +1,226 @@
+"""
+The MIT License (MIT)
```

Comment:
Is this a direct copy of code found elsewhere? If it is not, and is instead a derivative work thereof, I suggest we license this under the AGPL, like the rest of the XBlock, and in this comment acknowledge copyright of code this is based on, and that the MIT license applies to _that_ code.

Reply:
Everything in the /guacamole directory is copied from https://github.com/mohabusama/pyguacamole/tree/master/guacamole. At this point I have only made a few changes that were _needed_ to fix errors that were coming up (mostly related to python2/3 issues) and I wasn't sure how to address the licence, but this makes sense, I'll do that.